### PR TITLE
Sync

### PR DESCRIPTION
**Pull request by Garlyle**

### CODE REVIEW

**Review by balintvecsey — CHANGES_REQUESTED** [review]
resolve conflicts

File: app/src/main/java/com/greenfox/opal/gitinder/model/response/Match.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    }
+
+    public long getMatched_at() {
+        return matched_at;
+    }
+
+    public ArrayList<String> getMessages() {
+        return messages;
+    }
+}
```
Comment (dorinagy) [inline app/src/main/java/com/greenfox/opal/gitinder/model/response/Match.java]: @Garlyle use 2 spaces for indentation pls

File: app/src/main/java/com/greenfox/opal/gitinder/model/response/MatchesResponse.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    }
+
+    public MatchesResponse(String message) {
+        super("error", message);
+    }
+
+    public List<Match> getMatches() {
+        return matches;
+    }
+}
```
Comment (dorinagy) [inline app/src/main/java/com/greenfox/opal/gitinder/model/response/MatchesResponse.java]: @Garlyle indentation

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -27,7 +27,13 @@
...
+        <activity android:name=".LoginActivity"></activity>
+
+        <receiver android:name=".service.MatchesBroadcast" >
+            <intent-filter>
+                <action android:name="android.intent.action.BOOT_COMPLETED" />
+            </intent-filter>
+        </receiver>
     </application>
 
 </manifest>
```
Comment (adamgyulavari) [inline app/src/main/AndroidManifest.xml]: @Garlyle new line at the end of file

File: app/src/main/java/com/greenfox/opal/gitinder/service/MatchesBroadcast.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+import retrofit2.Response;
+
+public class MatchesBroadcast extends BroadcastReceiver {
+  @Inject
+  ApiService service;
+
+  @Override
+  public void onReceive(Context context, Intent intent) {
+    GitinderApp.app().basicComponent().inject(this);
+    service.getMatches("X-GiTinder-token").enqueue(new Callback<MatchesResponse>() {
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/opal/gitinder/service/MatchesBroadcast.java]: @Garlyle use the token stored in the shared preferences here

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/greenfox/opal/gitinder/service/MatchesBroadcast.java (hunk 4 above)
Comment (Garlyle) [reply app/src/main/java/com/greenfox/opal/gitinder/service/MatchesBroadcast.java]: @adamgyulavari changed

**Review by adamgyulavari — APPROVED** [review]

**Comment by adamgyulavari** [comment]
@Garlyle resolve conflicts

**Review by balintvecsey — APPROVED** [review]

**Pull request merged**